src/dlsnode/util/aio/internal/AioScheduler.d

```diff
@@ -60,7 +60,7 @@ class AioScheduler: ISelectEvent
 
     ***************************************************************************/
 
-    private TreeQueue!(SuspendableRequestHandler)[2] queues;
```

Comment:
Aside: it occurs to me that it might make more sense to rename `SuspendableRequestHandler` to `ContextWaitingForJob` or something like that.

Reply:
Yeah, this makes more sense/it's more generic. I'll address both issues (this and the `union` grouping) in the next PRs.